prime-router/src/test/kotlin/fhirengine/utils/FhirTranscoderTests.kt

```diff
@@ -56,8 +56,8 @@ class FhirTranscoderTests {
         """.trimIndent()
         val decodedBundle = FhirTranscoder.decode(encodedBundle)
         assertThat(decodedBundle).isNotNull()
-        assertThat(decodedBundle.id).equals("someid")
-        assertThat(decodedBundle.type).equals(Bundle.BundleType.MESSAGE)
+        assertThat(decodedBundle.id).isEqualTo("Bundle/someid")
```

Comment:
I'm genuinely unsure how this was passing previously, I can't get it to evaluate to true with the old code with debugging. It needs the `Bundle/`

Reply:
It is because it was using .equals vs .isEqualTo. .equals is always a pass

Reply:
Wow, I didn't realize .equals always passed. Thank you!